src/scripts/routes.js

```diff
@@ -391,17 +373,18 @@ define([
         roles: "admin",
         controller: "userprofilespage"
     });
+
     defineRoute({
         path: "/wizardremoteaccess.html",
         autoFocus: false,
         anonymous: true,
-        controller: "wizardremoteaccess"
+        controller: "wizard/access"
```

Comment:
I would name the controller `wizard/remoteaccess`, as it's a different thing than just `access` IMHO